### PR TITLE
Math-Magician

### PR DESCRIPTION
**Pull request by AhmadiGu**
### Test part for project

- add tests for calculate 
- add tests for operate
- add test for Home page
- add test for Quote page 
- add test for Calculator

### CODE REVIEW

File: src/test/Calculator.test.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import Calculator from '../components/Calculator';
+
+it('renders correctly', () => {
+  const tree = renderer.create(<Calculator />).toJSON();
+  expect(tree).toMatchSnapshot();
+});
```
Comment (DammyShittu) [inline src/test/Calculator.test.js]: - [ ] Good job including snapshot tests for the Calculator component, however you need to add tests simulating user interactions using the React Testing Library. Kindly add them.

  You can visit these links for idea on how to simulate user actions:
  -  [https://testing-library.com/docs/react-testing-library/intro/](https://testing-library.com/docs/react-testing-library/intro/)
  - [https://www.robinwieruch.de/react-testing-library/](https://www.robinwieruch.de/react-testing-library/)

**Review by DammyShittu — CHANGES_REQUESTED** [review]
Hi @AhmadiGu @krisbobo,

## Highlights
✔️ Descriptive PR
✔️ Passing linters
✔️ Good branch name

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/test/Calculator.test.js (hunk 1 above)
Comment (krisbobo) [reply src/test/Calculator.test.js]: Thanks for the links

Comment (Abenezer-Tilahun) [reply src/test/Calculator.test.js]: Fixed.

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @AhmadiGu 
 
Your project is complete  👍 💯   There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct Git-Flow.
✔️ Professional README  
✔️ Good Commit Messages
✔️ All Test Are Passing


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by krisbobo** [comment]
Good job @AhmadiGu

**Pull request merged**